packages/codegen-ui/lib/generate-form-definition/generate-form-definition.ts

```diff
@@ -72,5 +77,9 @@ export function generateFormDefinition({
 
   formDefinition.form.layoutStyle = mapStyles(form.style);
 
+  if (form.ctaConfig) {
```

Comment:
if this is an empty object it will be true. is that intentional?

Reply:
Yes empty objects will be allowed.  Added a test to confirm expectations.